rollup.config.js

```diff
@@ -17,7 +17,7 @@ import postcss from 'rollup-plugin-postcss'
 
```

Comment:
This file _should_ be moved to `src/crkeng/frontend`, but that can wait for another PR 😅